src/main/scala/org/broadinstitute/dsde/firecloud/webservice/CromIamApiService.scala

```diff
@@ -25,7 +23,9 @@ trait CromIamApiService extends HttpService with FireCloudRequestBuilding with F
       path("query") {
         pathEnd {
           get {
-            passthrough(s"$workflowRoot/query", HttpMethods.GET)
+            extract(_.request.uri.query) { query =>
+              passthrough(Uri(s"$workflowRoot/query").withQuery(query), HttpMethods.GET)
```

Comment:
My "DRY" senses are tingling... Presumably this is a common enough pattern that it might be worth making a `passthroughWithQueryParams` alternative to `passthrough`?

Reply:
I don't know why passthrough doesn't just do the right thing... who would think passthrough would not include the query parameters?

Reply:
It was certainly a surprise to all of us....

Reply:
Maybe if `passthrough` is our own custom function (rather than something predefined in the dsl), maybe we can rename it to `passthroughWithoutQueryParams`...?

Reply:
I think the basic flaw in `passthrough()` is that it accepts a `spray.http.Uri` instead of a `spray.routing.RequestContext#request`.

We could add a version that takes the request instead and automatically passes through the query params, but we won't be much safer until we remove the old query-dropping version... which entails changing 70+ code locations.

Since Orch is almost EOL I didn't think it was worth it.

Reply:
But now that I've thought through the whole thing in a detailed manner maybe I'll just do it.